client/src/components/AvailableDeskList.tsx

```diff
@@ -0,0 +1,63 @@
+import { List } from "@mui/material";
+import { useState, FC, useEffect, useContext } from "react";
+import { DeskType, Booking } from "../types";
+import { DeskItem } from "./DeskItemButton";
+import { FormContext, getUnixTime } from "../pages/bookingPage";
+import { isBetween } from "../helperFunctions";
+
+export const AvailableDeskList: FC = () => {
+  const [currentDeskList, setCurrentDeskList] = useState([] as DeskType[]);
+  const {
+    startTime: [startTimeValue],
+  } = useContext(FormContext);
+  const {
+    endTime: [endTimeValue],
+  } = useContext(FormContext);
+  const {
+    date: [dateValue],
+  } = useContext(FormContext);
+
+  useEffect(() => {
+    const setDeskList = async () => {
+      const bookingList = await fetch(
+        `${process.env.REACT_APP_ROOT_URL}api/bookinglist`
+      );
+      const jsonBookingList = await bookingList.json();
+      const bookings: Booking[] = jsonBookingList.bookingList;
+
+      const bookingsAtTime = bookings.filter((booking: Booking) => {
+        const unixStartTime = getUnixTime(dateValue, startTimeValue);
```

Comment:
It would be better to move the getUnixTime helper from the bookingPage component to some kind of util file.